.github/pull_request_template.md

```diff
@@ -18,21 +18,7 @@ All Pull Requests must have an issue attached to them
 
 Resolves #
 
-## Target Release
 
-<!--
-
-In normal circumstances we only target changes at the upcoming minor
-release, or as a patch to the current minor version. If you need to
-port a security fix to an older release, highlight this here by listing
-all targeted releases.
-
-If targeting the next patch release, also add the relevant x.y-backport
```

Comment:
I believe only this part is relevant to the backport bot? The remaining part seems to be a convention to indicate which versions this PR should go to, which does still make sense to me. WDYT?